src/installation/guides/chroot.md

```diff
@@ -0,0 +1,313 @@
+# Installation via chroot (x86 or x86_64)
+
+This guide details the process of manually installing Void Linux (the chroot
+method) on an x86 or x86_64 PC architecture. It is assumed that you have a
+familiarity with Linux but not necessarily that you have performed a chroot
+installation before. If you are unfamiliar with the process, this guide can be
+followed literally to create a "typical" setup (one partition is used on one
+disk that is connected by SATA, IDE or USB). If you are comfortable with the
+process, this guide should serve as an outline for reference, where each step
+may be modified to create less typical setups (such as [full disk
+encryption](./fde.md)).
+
+Void Linux provides two options for bootstrapping the new installation.
+Following the **XBPS method**, the [XBPS Package Manager](../../xbps/index.md)
+running on the host operating system installs the base distribution. Following
+the **ROOTFS method**, the base operating system is installed by unpacking a
+ROOTFS tarball available to download.
+
+The **XBPS method** requires that the host operating system have XBPS installed.
+This may be an already working installation of Void Linux, an official [live
+image](../live-images/prep.md) or a [statically linked
+XBPS](../../xbps/troubleshooting/static.md) running on any working Linux
+installation.
+
+The **ROOTFS method** requires only a host operating system that can enter a
+Linux chroot and has [tar(1)](https://man.voidlinux.org/tar.1) and
+[xz(1)](https://man.voidlinux.org/xz.1) installed. This method may be preferable
+if you wish to install Void Linux using a different Linux distribution.
+
+## Prepare Filesystems
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8) is also available to
+create FAT32 partitions. However, due to restrictions associated with FAT
+filesystems, it should only be used when no other filesystem is suitable (such
+as the EFI System Partition).
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on the live images
+for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+For a UEFI booting system, make sure to create an EFI System Partition (ESP).
+The ESP should have the partition type "EFI System" (code `EF00`) and be
+formatted as fat32 using [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+If unsure what partitions to create, create (1) 1GB in size with type `EFI
```

Comment:
"If you're unsure what partitions to create, create a 1GB partition of type ..."

Reply:
yes